project/src/components/place-card/place-card.tsx

```diff
@@ -24,14 +35,16 @@ function PlaceCard(): JSX.Element {
         </div>
         <div className="place-card__rating rating">
           <div className="place-card__stars rating__stars">
-            <span style={{width: '80%'}}></span>
+            <span style={{width: `${offer.rating * 100 / 5}%`}}></span>
             <span className="visually-hidden">Rating</span>
           </div>
         </div>
         <h2 className="place-card__name">
-          <a href="/#">Beautiful &amp; luxurious apartment at great location</a>
+          <Link to={AppRoute.Room + offer.id} onClick={() => history.push(AppRoute.Room + offer.id)}>
```

Comment:
Маршрут должен быть вида /offer/5 а не /offer/:id/5 onClick надо удалить, линк и так сделает переход по нужному адресу
```suggestion
          <Link to={AppRoute.Room + offer.id}}>
```